example/apache-ignite/src/main/scala/org/finos/vuu/example/ignite/IgniteOrderStore.scala

```diff
@@ -87,6 +87,16 @@ class IgniteOrderStore(private val parentOrderCache: IgniteCache[Int, ParentOrde
 
   }
 
+  def getCount(sqlFilterQueries: String): Long = {
+    //todo should this be COUNT_BIG?
+    val whereClause = if(sqlFilterQueries == null || sqlFilterQueries.isEmpty) "" else s" where $sqlFilterQueries"
+    val query = new SqlFieldsQuery(s"select COUNT(1) from ChildOrder$whereClause")
+    val cursor = childOrderCache.query(query)
+
+    val countValue = cursor.getAll().get(0).get(0)
```

Comment:
Would we need to put a guard around this? I guess not as there should always be a count (even if its zero)